### PR TITLE
Account page FE

### PR DESCRIPTION
**Pull request by haneefghanim**
Created the FE for the account page. Hooked up to fake data for now, need to pull in Esco's work on the thorchain bridge and integrate that into the FE.

### CODE REVIEW

**Comment by haneefghanim** [comment]
@Jessicawatsonmiller @philipstanislaus Is there any way to mark CodeClimate issues as invalid? It seems to think that a couple unrelated styled components are duplicated code issues.

**Comment by haneefghanim** [comment]
screen shot 2018-11-21 at 12 39 28 am

Screenshot attached here.

**Comment by Jessicawatsonmiller** [comment]
> @Jessicawatsonmiller @philipstanislaus Is there any way to mark CodeClimate issues as invalid? It seems to think that a couple unrelated styled components are duplicated code issues.

Yeah, I checked out the issues; no idea why it's doing that. We can ignore it for now and I'll just put a note on the CodeClimate review for this PR. We can look into CC settings later to prevent this in the future.

**Comment by Jessicawatsonmiller** [comment]
> Code Climate has analyzed commit [a0f0e6a](https://github.com/thorchain/ASGARDEX/commit/a0f0e6a80319b9377b1d3d565e3bf5a013db3a3b) and detected **4 issues** on this pull request.
> 
> Here's the issue category breakdown:
> 
> Category	Count
> Duplication	4
> View more on [Code Climate](https://codeclimate.com/github/thorchain/ASGARDEX/pull/6).

These are unrelated styled components; there is no need to refactor.

File: gui/src/organisms/AccountOrdersList.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import * as moment from 'moment'
+import * as React from 'react'
+import styled from 'styled-components'
+import { Header } from '../atoms/Header'
+import { Table, TableBody, TableCell, TableHeader, TableRow } from '../atoms/Table'
+
+// TODO replace with real orders data
```
Comment (Jessicawatsonmiller) [inline gui/src/organisms/AccountOrdersList.tsx]: IIRC this is out of scope for this task, and will be done in HEL03 or HEL04, or possibly pushed back as it's low priority. Dummy data is fine for now.

File: gui/src/organisms/AccountSetup.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            <Text>
+              10 ETH
+            </Text>
+            <Text>
+              1 BTC
+            </Text>
+          </Col>
+          <Col>
+            <Label>Address</Label>
+            <Text>TOX12345.....ABCDEF</Text>
```
Comment (Jessicawatsonmiller) [inline gui/src/organisms/AccountSetup.tsx]: So these text fields will be  replaced by the data from Esco's wallet bridge?

**Comment by Jessicawatsonmiller** [comment]
Looks great; can't wait to integrate it with the wasm bridge.

**Comment by Jessicawatsonmiller** [comment]
> screen shot 2018-11-21 at 12 39 28 am
> 
> Screenshot attached here.

Can we replace 'Unlock wallet' in the top right hand corner with 'Add Wallet' to make it clearer?

File: gui/src/organisms/AccountOrdersList.tsx (hunk 1 above)
Comment (haneefghanim) [reply gui/src/organisms/AccountOrdersList.tsx]: 👍

File: gui/src/organisms/AccountSetup.tsx (hunk 2 above)
Comment (haneefghanim) [reply gui/src/organisms/AccountSetup.tsx]: Yep, will be working on that today.
